### ISSUE TITLE
bug(a11y): dashboards not accessible via keyboard only navigation

### ISSUE THREAD

**Issue by donroyco** [theodo/falco#66]
**Reporting**
✅ bug
⬜️ feature
⬜️ something else

**Current situation**
Navigate to https://falco.theo.do/project and land on the home page. When tabbing through the content it skips the three dashboards (TTI, Speed Index, Load Time).

![GIF of current behaviour](https://user-images.githubusercontent.com/1763537/67631884-84e99d00-f89c-11e9-9f17-894077e82fcb.gif)
_Above GIF is a clip of Mac VoiceOver so the outlines are clearly visible._

`Add / Delete metrics →` is even not visibly focussed when tabbing through the content.

![GIF of non visible focus of button](https://user-images.githubusercontent.com/1763537/67632068-b2374a80-f89e-11e9-8d02-9f3ab046f514.gif)
_GIF of tabbing through content, `Add / Delete metrics →` is not visibly focussed._

**Expected situation**
I would expect that using the keyboard or using [Mac's built in VoiceOver](https://help.apple.com/voiceover/mac/10.15/#) would go through all the content of the page, so the content is accessible for all users.

**Comment by phacks** [comment]
@donroyco This is a problem indeed. My guess is that we messed up a `<div>` for a `<button>` or an `<a>`. We’re going to have a deeper look into accessibility in the upcoming week, although I think this one can be fixed earlier.

I can take care of this, however should you want to have a look at the code first and try to fix this, I’d be happy to help you navigate the codebase!

**Comment by fargito** [comment]
I think this issue is fixed in https://github.com/theodo/falco/pull/161
